src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$days: 31;
+$month-length: 28, 29, 30, 31;
+$start-days: (
+  'mon': 0,
+  'tue': 101px,
+  'wed': 202px,
+  'thu': 303px,
+  'fri': 404px,
+  'sat': 505px,
+  'sun': 606px,
+);
+
+html {
+  height: 100%;
+}
+
 body {
+  height: 100%;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 0;
+  list-style: none;
+  cursor: pointer;
+
+  @each $start-day, $offset in $start-days {
+    &--start-day-#{$start-day} .calendar__day--1 {
+      margin-left: $offset;
+    }
+  }
+
+  @for $i from 1 through 3 {
+    &--month-length-#{31 - $i} :nth-last-child(-n + #{$i}) {
+      display: none;
+    }
+  }
```

Comment:
The loop in lines 42-46 is intended to hide the last few days in months that are shorter than 31 days. However, the loop only runs through 3 iterations. Make sure this loop accounts for all possible month lengths defined in the `$month-length` variable.

Reply:
Here is no need to iterate over all days. Minimum month days length is 28 in February. Your comment is not relevant.